manifests/z/Zoom/Zoom/OutlookPlugin/5.6.6.153/Zoom.Zoom.OutlookPlugin.yaml

```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Zoom.Zoom.OutlookPlugin
+PackageVersion: 5.6.6.153
```

Comment:
```suggestion
PackageVersion: 5.6.6
```

Reply:
How would this work if they ever release a version 5.6.6.154+?

Just override winget version 5.6.6 with the new subversion?

Reply:
Ahh i didn't see this reply, i have no idea, it might be the same version just packed for each release, seems like you just have to update this manifest for each release if its different... We could check the hash/signature date on the next release.